Common/UnitDefinitions/MolarFlow.json

```diff
@@ -0,0 +1,107 @@
+{
+  "Name": "MolarFlow",
+  "BaseUnit": "KilomolePerSecond",
+  "XmlDocSummary": "Molar flow is the ratio of the amount of substance change to the time during which the change occurred (value of amount of substance changes per unit time).",
+  "BaseDimensions": {
+    "N": 1,
+    "T": -1
+  },
+  "Units": [
+    {
+      "SingularName": "KilomolePerSecond",
+      "PluralName": "KilomolesPerSecond",
+      "BaseUnits": {
+        "N": "Kilomole",
+        "T": "Second"
+      },
+      "FromUnitToBaseFunc": "{x}",
+      "FromBaseToUnitFunc": "{x}",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kmol/s" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilomolePerMinute",
+      "PluralName": "KilomolesPerMinute",
+      "BaseUnits": {
+        "N": "Kilomole",
+        "T": "Minute"
+      },
+      "FromUnitToBaseFunc": "{x} / 5.999988e1",
+      "FromBaseToUnitFunc": "{x} * 5.999988e1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kmol/min" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilomolePerHour",
+      "PluralName": "KilomolesPerHour",
+      "BaseUnits": {
+        "N": "Kilomole",
+        "T": "Hour"
+      },
+      "FromUnitToBaseFunc": "{x} / 3.599997e3",
+      "FromBaseToUnitFunc": "{x} * 3.599997e3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kmol/h" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundMolePerSecond",
```

Comment:
I was unsure about PoundMole vs Poundmole, but it seems "pound mole" or "pound-mole" is how it is typically written, so PoundMole is probably more consistent.
No change needed.